test/test_client.py

```diff
@@ -1629,7 +1629,23 @@ def test_service_name_from_kwargs(self):
              connect=False)
         self.assertEqual(client._topology_settings._srv_service_name,
                          'customname')
-
+    def test_srv_max_hosts(self):
```

Comment:
What about all the other new tests added in https://github.com/mongodb/specifications/pull/1069?

Reply:
I will resync spec tests.